tests/e2e/default/wdio.conf.js

```diff
@@ -44,9 +43,11 @@ const baseConfig = {
   // then the current working directory is where your `package.json` resides, so `wdio`
   // will be called from there.
   //
-  specs: [
-    '**/*.wdio-spec.js',
-  ],
+  
+  suites: {
+    all: ['**/*.wdio-spec.js'],
+    ...wdiUtils.suites,
+  },
```

Comment:
I think we should move this conf file up one level as it's the base config for all. Then we can make a new `default/wdio.conf.js` which extends the base and just defines the specs/suites.

Doesn't need to be done in this PR but felt like it fitted.

Reply:
Good point @garethbowen . I have pushed it up